nav2_costmap_2d/plugins/static_layer.cpp

```diff
@@ -120,9 +120,9 @@ void StaticLayer::onInitialize()
   }
 
   dynamic_param_client_ = new nav2_dynamic_params::DynamicParamsClient(node_);
+  dynamic_param_client_->add_parameters({"enabled_static_layer"});
```

Comment:
It would be nice to have an `add_parameters` variant that takes a string, so we can drop the initializer list in the case of a single parameter.

Reply:
Added the variant in latest commit. When overloading with a string argument, I was getting that the call was ambiguous (assuming users call function like `add_parameters("enabled_static_layer")` and `add_parameters({"foo", "bar"})`) so I ended up passing a character array instead.